scripts/release.js

```diff
@@ -56,8 +56,6 @@ const yargs = require('nx/node_modules/yargs');
 
     const { workspaceVersion, projectsVersionData } = await releaseVersion({
       specifier: options.version,
-      // stage package.json updates to be committed later by the changelog command
-      stageChanges: true,
```

Comment:
This is essentially always needed for the common case, so it's true by default now